src/modules/sdk/objections/service.ts

```diff
@@ -48,13 +53,21 @@ export const patchObjection = async (
   await makeAPICall(axiosConfig);
 };
 
-export const addAttachment = (companyNumber: string,
-                              token: string,
-                              objectionId: string,
-                              attachment: Buffer,
-                              fileName: string) => {
+export const addAttachment = async (companyNumber: string,
+                                    token: string,
+                                    objectionId: string,
+                                    attachment: Buffer,
+                                    fileName: string) => {
+
+  const axiosConfig: AxiosRequestConfig = getBaseAxiosRequestConfig(
+    HTTP_POST,
+    OBJECTIONS_API_ADD_ATTACHMENT_URL(companyNumber, objectionId),
```

Comment:
Is there going to be any error handling here or are we doing that a later date on another ticket?

Reply:
makeApiCall will log and re-throw the error. Best to let it propagate back to the controller where it can deal with it by calling next(e)